tg_bot/modules/locks.py

```diff
@@ -80,6 +83,59 @@ def locktypes(bot: Bot, update: Update):
     update.effective_message.reply_text("\n - ".join(["Locks: "] + list(LOCK_TYPES) + list(RESTRICTION_TYPES)))
 
 
+@user_admin
+def add_whitelist(bot: Bot, update: Update):
+    chat = update.effective_chat  # type: Optional[Chat]
+    message = update.effective_message  # type: Optional[Message]
+    entities = message.parse_entities(MessageEntity.URL)
+    added = []
+    for url in entities.values():
+        if sql.add_whitelist(chat.id, url):
+            added.append(url)
+    if added:
+        message.reply_text("Added {} to whitelist.".format(', '.join(w for w in added)))
```

Comment:
Unnecessary list comprehension.
Also, consider using newlines starting with `-` to delimit the different urls; putting everything on one line will be messy

Reply:
Agree, fixed.